cinema/models.py

```diff
@@ -39,8 +43,18 @@ class Movie(models.Model):
     title = models.CharField(max_length=255)
     description = models.TextField()
     duration = models.IntegerField()
-    genres = models.ManyToManyField(Genre)
-    actors = models.ManyToManyField(Actor)
+    genres = models.ManyToManyField("Genre")
+    actors = models.ManyToManyField("Actor")
+
+    def movie_image_path(self, filename):
+        ext = filename.split(".")[-1]
+        filename = f"{slugify(self.title)}-{uuid.uuid4()}.{ext}"
+        return os.path.join("uploads/movies/", filename)
```

Comment:
better make that function separate from model or use a staticmethod as you don't use `self`